docs/eventing/sources/rabbitmq-source/README.md

```diff
@@ -0,0 +1,219 @@
+        kind: RabbitmqCluster
+        metadata:
+          name: <cluster-name>
+          annotations:
+            # A single RabbitMQ cluster per Knative Eventing installation
+            rabbitmq.com/topology-allowed-namespaces: "*"
+        ```
+        Where `<cluster-name>` is the name you want for your RabbitMQ cluster,
+        for example, `rabbitmq`.
+
+    1. Apply the YAML file by running the command:
+
+        ```bash
+        kubectl create -f <filename>
+        ```
+        Where `<filename>` is the name of the file you created in the previous step.
+
+1. Wait for the cluster to become ready. When the cluster is ready, `ALLREPLICASREADY`
+will be `true` in the output of the following command:
+
+    ```bash
+    kubectl get rmq <cluster-name>
+    ```
+    Where `<cluster-name>` is the name you gave your cluster in the step above.
+
+    Example output:
+
+    ```{ .bash .no-copy }
+    NAME          ALLREPLICASREADY   RECONCILESUCCESS   AGE
+    rabbitmq      True               True               38s
+    ```
+
+For more information about configuring the `RabbitmqCluster` CRD, see the
+[RabbitMQ website](https://www.rabbitmq.com/kubernetes/operator/using-operator.html).
+
+## Reference an external RabbitMQ Cluster
+
+1. Create a Secret that stores the external RabbitMQ Cluster Credentials and URI:
+
+    ```yaml
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: rabbitmq-secret-credentials
+    # This is just a sample, don't use it this way in production
+    stringData:
+      username: $EXTERNAL_RABBITMQ_USERNAME
+      password: $EXTERNAL_RABBITMQ_PASSWORD
+      uri: $EXTERNAL_RABBITMQ_MANAGEMENT_UI_URI:$PORT
+    ```
+
+You will reference this Secret in the RabbitMQ Source object later.
+
+## Create a Service
+
+1. Create the `event-display` Service as a YAML file:
+
+     ```yaml
+     apiVersion: serving.knative.dev/v1
+     kind: Service
+     metadata:
+       name: event-display
+       namespace: default
+     spec:
+       template:
+         spec:
+           containers:
+             - # This corresponds to
+               # https://github.com/knative/eventing/tree/main/cmd/event_display/main.go
+               image: gcr.io/knative-releases/knative.dev/eventing/cmd/event_display
+     ```
+
+1. Apply the YAML file by running the command:
+
+    ```bash
+    kubectl apply -f <filename>.yaml
+    ```
+    Where `<filename>` is the name of the file you created in the previous step.
+
+    Example output:
+    ```{ .bash .no-copy }
+    service.serving.knative.dev/event-display created
+    ```
+
+1. Ensure that the Service Pod is running, by running the command:
+
+    ```bash
+    kubectl get pods
+    ```
+
+    The Pod name is prefixed with `event-display`:
+    ```{ .bash .no-copy }
+    NAME                                            READY     STATUS    RESTARTS   AGE
+    event-display-00001-deployment-5d5df6c7-gv2j4   2/2       Running   0          72s
+    ```
+
```

Comment:
this should be a snippet - if it's not already, let's make it one and then include it here instead of having another duplicate section on creating services that needs to be maintained